app/controllers/idv/phone_errors_controller.rb

```diff
@@ -45,5 +46,14 @@ def track_event(type:)
 
       analytics.idv_phone_error_visited(**attributes)
     end
+
+    # rubocop:disable Naming/MemoizedInstanceVariableName
+    def set_gpo_letter_available
+      @gpo_letter_available ||= FeatureManagement.enable_gpo_verification? &&
```

Comment:
Same thing existed previously, but won't this reassign the instance variable if called repeatedly while still `false`?

Should it be... ?

```suggestion
      @gpo_letter_available &&= FeatureManagement.enable_gpo_verification? &&
```

Reply:
Yep, happy to fix!

Reply:
I ended up going with this because of some quirks with the `&&=` operator:

```
[1] pry(main)> a &&= true
=> nil
[2] pry(main)> a
=> nil
```